tests/integration/scripts/test_broadcast.py

```diff
@@ -77,3 +78,59 @@ async def test_broadcast_multi(
                     ' settings are not compatible with the workflow'
                 ) in out
                 assert err == ''
+
+
+async def test_broadcast_multi_namespace(
+    flow,
+    scheduler,
+    start,
+    db_select,
+):
+    """Test a multi-namespace broadcast command.
+
+    See https://github.com/cylc/cylc-flow/issues/6334
+    """
+    id_ = flow(
+        {
+            'scheduling': {
+                'graph': {'R1': 'a & b & c & fin'},
+            },
+            'runtime': {
+                'root': {'execution time limit': 'PT1S'},
+                'VOWELS': {'execution time limit': 'PT2S'},
+                'CONSONANTS': {'execution time limit': 'PT3S'},
+                'a': {'inherit': 'VOWELS'},
+                'b': {'inherit': 'CONSONANTS'},
+                'c': {'inherit': 'CONSONANTS'},
+            },
+        }
+    )
+    schd = scheduler(id_)
+
+    async with start(schd):
+        # issue a broadcast to multiple namespaces
```

Comment:
I managed to replicate the error doing this:

```bash
$ cylc broadcast throwaway/wide.dirt -p '*' -n baz -n bar -n root -s 'script=sleep 30'  -s '[simulation]default run length = PT0S'
ERROR: [{'error': {'message': "'DurationFloat' object has no attribute 'startswith'", 'traceback': ["graphql.error.located_error.GraphQLLocatedError: 'DurationFloat' object has no attribute 'startswith'\n"]}}]
```

Perhaps you should test that too?